packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpMvsApi.ts

```diff
@@ -141,7 +150,27 @@ export class FtpMvsApi extends AbstractFtpApi implements ZoweExplorerApi.IMvs {
                     throw new Error();
                 }
             }
-            await DataSetUtils.uploadDataSet(connection, targetDataset, transferOptions);
+            const lrecl: number = dsAtrribute.apiResponse.items[0].lrecl;
+            const data = fs.readFileSync(inputFilePath, "UTF-8");
+            const lines = data.split(/\r?\n/);
+            const foundIndex = lines.findIndex((line) => line.length > lrecl);
+
+            if (foundIndex !== -1) {
+                const select = await vscode.window.showWarningMessage(
+                    "zftp Warning: The " +
```

Comment:
At least one line, like line 10, is longer than dataset LRECL, 80. The exceeding part will be truncated. Do you want to continue?

Reply:
Thank you @std4lqi ! I updated the message and the variable name.